package.json

```diff
@@ -48,14 +48,14 @@
     "gulp-rename": "^1.2.2",
     "gulp-sourcemaps": "^1.6.0",
     "gulp-uglify": "*",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "react": "^16.3.0",
+    "react-dom": "^16.3.0",
     "vinyl-buffer": "^1.0.0",
     "vinyl-source-stream": "^1.1.0"
   },
   "peerDependencies": {
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0"
+    "react": "^16.3.0",
+    "react-dom": "^16.3.0"
   },
   "dependencies": {
     "babel-preset-es2015": "^6.16.0",
```

Comment:
@ZachTRice I think we want to replace the dependency `babel-preset-es2015` here with `babel-preset-env`, right?

Reply:
`babel-preset-env` is in devDependency. We can remove `babel-preset-es2015` though, and, I'm assuming `babel-preset-react` could move up into devDependencies